### ISSUE TITLE
Unused url dependency

### ISSUE THREAD

**Issue by screendriver** [depcheck/depcheck#187]
### index.js

```js
import url from 'url';

export async function foo() {
  // ...
  const bar = url.resolve('...', '...');
  // ...
}
```

### package.json

```js
"dependencies": {
  "url": "0.11.0"
}
```

### depcheck `v0.6.6`

```sh
Unused dependencies
* url
```

I am using Node.js `v7.2.0`

**Comment by lijunle** [comment]
This is strange, what is the output of `depcheck --json`?

**Comment by dsernst** [comment]
I'm seeing this as well in a `react-native` project.

**Comment by dsernst** [comment]
Depcheck error if `url` is in package.json dependencies:
```
19:57 $ depcheck . --ignores=buildStyleInterpolator,eslint-* --ignore-dirs=ios
Unused dependencies
* url
```

--------------------

React native fatal error after `npm rm -S url`:
>

<img src="https://cloud.githubusercontent.com/assets/6340841/24318933/02abcc60-10cc-11e7-9162-5f491362ed13.png" height="687px" />

**Comment by mnkhouri** [comment]
Thank you for the report!

Quoting from my [comment here](https://github.com/depcheck/depcheck/issues/251#issuecomment-416110283): 
> I've done a bit of investigation, and discovered that this false positive happens because `depcheck` filters out any node.js [built-in modules](https://github.com/sindresorhus/builtin-modules/blob/master/builtin-modules.json) from the list of "used modules" ([code to do so is here](https://github.com/depcheck/depcheck/blob/539c6dd/src/check.js#L145)).

I'm going to close this issue as a duplicate of https://github.com/depcheck/depcheck/issues/251, since I've added some brainstorming about possible solutions there
